public/index.html

```diff
@@ -1,38 +1,28 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+
+<head>
     <meta charset="utf-8" />
     <base href="/" />
-    <link rel="shortcut icon" href="favicon.ico" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1, shrink-to-fit=no"
-    />
+    <link rel="shortcut icon" href="Logo.png" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
     <!-- Primary Meta Tags -->
     <title>Meal Sharing App</title>
     <meta name="title" content="Meal Sharing App" />
-    <meta
-      name="description"
-      content="Website api for sharing meals together. Invite your best friends or your family for dinner!"
-    />
+    <meta name="description" content="Website api for sharing meals together. Invite your best friends or your family for dinner!" />
```

Comment:
A little detail but is it a website or an API? :D 
It's either a website (that happens to use an API)
Or it's an API (that can be consumed by a website, or a phone app or something else)